yaml/roads.yaml

```diff
@@ -235,14 +236,26 @@ filters:
         - {tags->ski: designated}
     min_zoom: 'LEAST(mz_calculate_path_major_route(osm_id), 13)'
     table: osm
+    extra_columns: [bicycle, foot, horse, tracktype, "tags->incline", "tags->trail_visibility", "tags->sac_scale"]
```

Comment:
Yup, they were moved to `output` in a323875e967b768c38c9a518034e729c1bb787df.

`extra_columns` is for columns which need to be available because they're used in an expression, but not used in output or any filter. This because the script which generates the SQL can't "see" inside the SQL expressions, and therefore has no way of knowing that a column is used there. If the column isn't used anywhere else, then you'll see errors saying that the variable isn't defined.

There _was_ only one place where it was needed [in `pois.yaml`](https://github.com/mapzen/vector-datasource/blob/master/yaml/pois.yaml#L194-L200), but it seems we're outputting `height` now, so the `extra_columns` statement is superfluous now.